src/digid_eherkenning_oidc_generics/migrations/0008_auto_20230109_1111.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.16 on 2023-01-09 10:11
```

Comment:
these shouldn't be here - instead you need to fix your local environment by updating the dependencies:

```bash
pip install -r requirements/dev.txt
src/manage.py migrate
```